lib/@uncharted/personas/src/layout/breadcrumbs/Breadcrumb.js

```diff
@@ -215,6 +205,25 @@ export class Breadcrumb extends CachedNode {
     }
 
     /**
+     * Updates this breadcrumb's icon.
+     *
+     * @method updateIcon
+     *
+     * @param {Layout} layout - The layout used to take a snapshot of.
```

Comment:
Sounds like a dangling preposition or some such.  Either a) drop "of", b) add a noun indicating what the snapshot is of.  the layout used to take a snapshot of the current level.  (or whatever it actually does), or c) something like "The layout of the snapshot" or "The layout to use for the snapshot"